app/config/errorCodes.php

```diff
@@ -0,0 +1,170 @@
+<?php
```

Comment:
I feel we should have a better pattern for handling prefixes for all the constants. We should probably document it as well.

Reply:
I have made an attempt at this. Please check and let me know